api/src/main/scala/dcos/metronome/api/v1/controllers/ApplicationController.scala

```diff
@@ -19,6 +21,10 @@ class ApplicationController(metrics: Metrics) extends RestController {
 
   def ping = Action { Ok("pong") }
 
+  def info = Action {
+    Ok(MetronomeInfoWrites.writes(MetronomeInfoBuilder.metronomeInfo))
```

Comment:
The writes is implicit, you shouldn't need to explicitly use it here.

(In case you did that to make it obvious where the conversion comes from, that's fine with me.)

Reply:
yeah we talked with Ken about this use of implicits and we agreed that we prefer doing these explicitly...

Reply:
That's what I imagined :)